test-utils/testlib/src/fees_utils.rs

```diff
@@ -176,7 +176,7 @@ impl FeeHelper {
     /// - The base cost of the delegate action (send and exec).
     /// - The additional send cost for all inner actions.
     #[cfg(feature = "protocol_feature_nep366_delegate_action")]
-    pub fn meta_tx_overhead_cost(&self, actions: &[Action]) -> Balance {
+    pub fn meta_tx_overhead_cost(&self, actions: &[Action], is_receiver_implicit: bool) -> Balance {
```

Comment:
Not strictly related, but can't we replace most of it with `total_send_fees` call, or its slight modification?
That function does exactly what we want - computes send fees for all actions - and reusing it could help to avoid the bug below at the first place. Also fees logic is hard to read in general, so the less code we have, the better.

Reply:
Very good point! Updated.